src/build/hash.js

```diff
@@ -0,0 +1,45 @@
+const crypto = require('crypto');
+const fs = require('fs');
+
+const hashLength = 8;
+
+/**
+ * Hashes the passed path. If the file doesn't exist, return a random hash. If
+ * we're in prod, then crash.
+ *
+ * @param {string} files to hash
+ * @return {string}
+ */
+function hashFor(...files) {
```

Comment:
Is this function used anywhere?

Reply:
No, it was leftover from the "resources" version in the other PR. I'll remove it for now.